lib/client/api_service.es6.js

```diff
@@ -53,11 +54,21 @@ angular.module('confluence').service('api', ['$window', function($window) {
       name: '/worker/apiConfluence'
     }, ctx),
   }, ctx);
+  const urlState = foam.web.URLState.create(null, ctx);
+  const bindState = function(object, name, defaultValue) {
+    const slot = pkg.PropertySlot.create({object, name});
+    if (foam.Undefined.isInstance(slot.get())) slot.set(defaultValue);
+    urlState.addBinding(name, slot);
+    return slot;
+  };
+  const clearState = function() { return urlState.clearBindings(); };
 
   return {
+    workerEvents,
     apiMatrixController,
     apiConfluence,
-    workerEvents: workerEvents,
+    bindState,
```

Comment:
Are these going to be used somewhere, or is there something automatic happening?

Reply:
Nope. Leftover from a previous (internal to this PR) iteration on this API. Fixed.